sos/options.py

```diff
@@ -70,7 +70,7 @@ def __str(self, quote=False, sep=" ", prefix="", suffix=""):
             vals = [",".join(v) if _is_seq(v) else v for v in vals]
         else:
             def is_string(val):
-                return isinstance(val, six.string_types)
+                return isinstance(val, str)
             # Only quote strings if quote=False
             vals = ["'%s'" % v if is_string(v) else v for v in vals]
```

Comment:
Maybe a silly question, but how much sense it has to have `is_string` method at all? Can't be here just:

```
vals = ["'%s'" % v if isinstance(v, str) else v for v in vals]
```

?

Reply:
You are correct, I have no idea why that is like that. Simplified.